### PR TITLE
Removed Intercom and Telligent

### PR DESCRIPTION
**Pull request by ryan-blunden**

### CODE REVIEW

File: website/src/pages/developer-platform/index.tsx [hunk 1]
```diff
@@ -233,7 +233,7 @@ export default ((props: any) => (
                         src="https://www.youtube.com/embed/KTJs-0EInW8"
                         frameBorder="0"
                         allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
-                        allowFullscreen={true}
+                        allowFullScreen={true}
```
[the remark this answers is not in the document]
Comment (ryan-blunden) [reply website/src/pages/developer-platform/index.tsx]: As I was checking different pages to confirm my code changes had not broken anything, I saw error messages in the console for invalid prop names.

File: docs/privacy.md [hunk 2]
```diff
@@ -75,9 +75,7 @@ We may use the data collected through cookies, log file, device identifiers, loc
 
 We use third-party automated devices and applications, such as Google Analytics, to evaluate usage of our Website. Google Analytics is a Web-based analysis tool provided by the Google Inc., 1600 Amphitheatre Pkwy, Mountain View, CA 94043-1351, USA. Google provides further information about its own privacy practices and offers a browser add-on to opt out of Google Analytics tracking at <a href="https://tools.google.com/dlpage/gaoptout">https://tools.google.com/dlpage/gaoptout</a>.
```
Comment (attfarhan) [inline docs/privacy.md]: Do you need to update "Last modified on" in this file? (On line 7, can't comment on that line in this diff)

File: website/src/EventLogger.tsx [hunk 3]
```diff
@@ -193,21 +193,6 @@ class EventLogger {
     }
 
     public trackEvent(category: string, action: string, feature: any, label: string, eventProps: object): void {
-        let props = { ...eventProps }
```
Comment (attfarhan) [inline website/src/EventLogger.tsx]: Are we going to re-use the EventLogger class in this file with the new event tracking system?

File: docs/privacy.md (hunk 2 above)
Comment (ryan-blunden) [reply docs/privacy.md]: Good catch and has been updated.

File: website/src/EventLogger.tsx (hunk 3 above)
Comment (ryan-blunden) [reply website/src/EventLogger.tsx]: I presume so, although I'm not aware of a new event tracking system.

Should we keep event props creation code, from `trackPageView` but just remove the Telligent code?

Comment (ryan-blunden) [reply website/src/EventLogger.tsx]: Confirmed with @ebrodymoore that GA as a data source is sufficient for the about site. At least for now.

**Pull request merged**